security/pkg/pki/util/generate_csr.go

```diff
@@ -45,7 +45,16 @@ func GenCSR(options CertOptions) ([]byte, []byte, error) {
 	if options.ECSigAlg != "" {
 		switch options.ECSigAlg {
 		case EcdsaSigAlg:
-			priv, err = ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+			var curve elliptic.Curve
+			switch options.ECCCurve {
+			case P384Curve:
+				curve = elliptic.P384()
+			case P256Curve:
+				fallthrough
```

Comment:
```suggestion
			case P256Curve:
				fallthrough
```

mega-nit: can we be consistent with default case + P256 default handling in all these case statements?

e.g. either 

```
case
case 256
  fallthrough
default
  256
```

OR

```
case
case 256
  256
default
  256
```

but not a mix-and-match?